### ISSUE TITLE
Shameless Twitter plug isn’t particularly welcome

### ISSUE THREAD

**Issue by thibaudcolas** [michaelgmcd/vscode-language-babel#72]
I think my extensions just auto-updated and I got the message added in 80b7c7af91874bdd8af823fb9849d653b6b6f429, 

> Thanks for downloading Language Babel! Make sure to follow me on Twitter @michaelgmcd

This isn’t something I’d like my editor’s extensions to display, ever. It might seem harmless to you but for me it feels like one more distraction that shows up out of nowhere, and if all extensions started doing this it would become full-on obnoxious. I don’t want to have to question whether messages displayed by extensions are always useful or plain spam.

---

If you want to promote your personal brand I would suggest to display this in the extension’s marketplace page. I’d be very happy to know more about the people behind extensions from there, __when I’m shopping for extensions__.

It would also be ok for this to be part of a more useful "the extension updated, please have a look at the release notes" like VS Code itself has, as long as there was a way to disable this too.

**Comment by michaelgmcd** [comment]
I understand your concern. This will only ever show up once, meaning future updates will not show it. For previous installs, it was on the auto-update and for new users, it's only on the initial install. I may remove it at some point in the near future, but I've spent 3+ years making and maintaining this for 650k+ users for free. I got the idea from the countless plugins that have a welcome message on install, and if there was a way to only show it for new users, I would have. I guess I could reword it to be something more general like you mentioned, but I thought the community would've embraced me asking for some simple, donation-less support. The backlash (1-star reviews and this issue) is pretty disheartening.

You're clearly active in the open-source community as well, so I thank you for that and for using my plugin. I'll leave this open to prevent others from creating the same issue. If people continue to have an issue with it, I'll remove it.

**Comment by thibaudcolas** [comment]
Hey @michaelgmcd, thanks for picking this up.

To be clear I think it’s good for you to ask for support / promote your work on this project, It’s just the channel that I find inappropriate. Personally I find it too obtrusive even just the one time, and if some extensions start doing this it creates a precedent that I don’t really like. Right now I have 36 extensions installed, and I use VS Code on 3 different computers. Just imagine if all the extensions started doing this, even just on install.

What do you think of adding this to the extension’s marketplace page / README instead? Here is an example of what this would look like in practice: https://marketplace.visualstudio.com/items?itemName=Equinusocio.vsc-material-theme#review-details.

[![material-theme](https://user-images.githubusercontent.com/877585/86234133-63a92100-bb8e-11ea-9ce0-75ba818593d8.png)](https://marketplace.visualstudio.com/items?itemName=Equinusocio.vsc-material-theme)

There are three means of promotion here – the "Supported by" badge in the README, the OpenCollective badge on the side, and the publisher’s name at the top, which can be clicked to go this page: https://marketplace.visualstudio.com/publishers/Equinusocio

![material-theme-publisher](https://user-images.githubusercontent.com/877585/86234357-b2ef5180-bb8e-11ea-9f9c-3a267185feec.png)

Anyway, this is just my take on this, others might see it differently.

**Comment by michaelgmcd** [comment]
I've decided to remove the Twitter message.
